src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -131,6 +133,22 @@ class JesBackend extends Backend with LazyLogging {
     JesOutput(value, engineFunctions.gcsPathFromAnyString(value).toString, Paths.get(value))
   }
 
+  override def stdout(workflowId: WorkflowId, callFqn: FullyQualifiedName): WdlFile = {
+    val fqnParts = callFqn.split("\\.")
```

Comment:
Completely agree w/ the FQN case class. The type being passed in should have already enforced any requirements. There should not be broken data flowing around.